evalml/pipelines/components/transformers/preprocessing/featuretools.py

```diff
@@ -42,6 +45,18 @@ def _make_entity_set(self, X):
             es = ft_es.add_dataframe(dataframe=X, dataframe_name="X", index=self.index)
         return es
 
+    def _filter_features(self, X, es):
+        features_to_use = []
+        for feature in self.features:
+            if not feature.entityset == es:
```

Comment:
@thehomebrewnerd here I'm using comparison of the feature's ES and the ES created using the input to judge if the columns needed by the feature exists. Is there a more direct method of looking at what columns a feature needs to simply this?

Reply:
@jeremyliweishih If you have a feature, you can access the other features it was built from by accessing `feature.base_features`. This will be a list - in a single table setup, I think this list will contain only `IdentifyFeatures`. From those base identify features you can access the dataframe name with `feature.dataframe_name`(if you need it) and the column name with `feature.column_name`.

So, putting this all together, to get a list of the columns used to create a feature, `f`,  in a single table entityset, you can do this:
```
inputs_cols = [f.column_name for f in f.base_features]
```

Things get a fair amount more complicated when you get to multi-table entitysets with an arbitrary feature depth, but that might be out of scope for what you are doing now.